hack/remove-calico-policy/README.md

```diff
@@ -0,0 +1,60 @@
+# Disabling and Removing Calico Policy
+
+This guide explains how to disable and remove Calico policy from a running cluster.  These steps are intended as a
+last resort should it appear that Calico policy is not functioning properly on a production system.
+
+The steps in this directory are specific to clusters running on GCE using the
+Kubernetes `cluster/kube-up.sh` script, but can be generalized to other Calico deployments.
+
+Calico policy can be disabled and removed for troubleshooting purposes or in emergency situations using the following steps.
+
+### Requirements / Assumptions
+
+- Calico version v2.2 or higher
+- Kubernetes v1.6 or higher
+- These steps assume Calico is running in policy-only mode (without BGP networking)
+
+### Building
+
+To build the `calico/iptables-remover` Docker image used by the DaemonSet:
+
+```
+docker build -t calico/iptables-remover .
+```
+
+### Instructions
+
+> **Note:** The following steps assume you're running [Calico on GCE via kube-up](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/calico-policy-controller)
+using the add-on manager.
+
+#### 1. Stop Felix on the nodes
+
+First, you must stop the Felix agent running on the nodes in question.  You can do this by removing the
+`projectcalico.org/ds-ready: "true"` label from the nodes.
+
+To remove from all nodes:
+
+```
+kubectl label nodes --all projectcalico.org/ds-ready-
+```
+
+Then, wait until all the `calico-node-xxxx` pods in the `kube-system` Namespace have terminated.
+
+#### 2. Remove any programmed policy from the nodes
+
+To remove any programmed policy from the nodes, you can deploy the DaemonSet in this directory.
+The DaemonSet runs on all nodes without the `projectcalico.org/ds-ready: "true"` label, and removes
+any Calico iptables rules on the node.
+
+The DaemonSet relies on a ConfigMap containing [the script to execute](remove-calico-policy.sh) from this
```

Comment:
I think we should move the config map step before the DaemonSet explanation since someone might read `To remove any programmed policy from the nodes, you can deploy the DaemonSet in this directory.` and might just run the DaemonSet (It's very likely if someone is following these steps they are at a critical point where the production cluster is not working :) )